Resources/Prototypes/Entities/Objects/Fun/toys.yml

```diff
@@ -298,6 +298,17 @@
   - type: Sprite
     state: plushie_lizard_mirrored
 
+- type: entity
+  name: remote lizard plushie
+  id: PlushieLizardRemote
+  parent: PlushieLizard
+  components:
+  - type: DeviceLinkSink
+    ports:
+      - Trigger
+  - type: TriggerOnSignal
+  - type: InteractOnTrigger
```

Comment:
There is already a comp for playing a sound on trigger, which this should be using instead of faking a hand interaction.

Reply:
Hmm. Yeah okay, I can make a BasePlushieLizard and rearrange the lizard plush yml since its collecting some members.

While you can also use the dedicated EmitSoundOnTrigger to emit sounds, you cannot use it for all Interactions. This is merely a demonstration entity.